webapp/portlet/src/main/webapp/idm-groups-management/components/GroupsManagementFormDrawer.vue

```diff
@@ -96,7 +96,7 @@ export default {
     saving: false,
     confirmNewPassword: null,
     group: {},
-    parentGroup: null,
+    parentGroup: null
```

Comment:
Just for curiosity, why this change ?

Reply:
in the first time, i add a popup and new variable popupBodyMessage and ali tell me that you don't need to add those so I revert code